src/main/java/thePackmaster/powers/serpentinepack/TaintedPower.java

```diff
@@ -0,0 +1,48 @@
+package thePackmaster.powers.serpentinepack;
+
+import com.evacipated.cardcrawl.mod.stslib.powers.interfaces.InvisiblePower;
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.animations.VFXAction;
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.actions.common.GainBlockAction;
+import com.megacrit.cardcrawl.actions.common.RemoveSpecificPowerAction;
+import com.megacrit.cardcrawl.actions.watcher.ChangeStanceAction;
+import com.megacrit.cardcrawl.actions.watcher.NotStanceCheckAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.localization.PowerStrings;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import com.megacrit.cardcrawl.vfx.combat.EmptyStanceEffect;
+import com.megacrit.cardcrawl.vfx.combat.SmokeBlurEffect;
+import thePackmaster.SpireAnniversary5Mod;
+import thePackmaster.powers.AbstractPackmasterPower;
+import thePackmaster.stances.serpentinepack.VenemousStance;
+
+public class TaintedPower extends AbstractPackmasterPower implements InvisiblePower {
```

Comment:
Why do you have this second invisible power here? You could've just used the OnReceivePower interface on the TaintedEnemy power